examples/accounts/scripts/run.sh

```diff
@@ -2,22 +2,30 @@
 
 set -e
 
+JSON="Content-Type: application/json"
+
+
+
 echo ""; echo "Step 1"; echo ""
-pact -a examples/accounts/scripts/01-system.yaml | curl -d @- http://localhost:8080/api/v1/send
-curl -d '{"requestKeys":["221b1ea8687ca43da7eacdedfffe5a590215c334482b1148d5166e9847ead25e4d3a9c711c85c0b5ab9b739f151b562f37e1e21f2c8bc1004a5765575069cd91"]}' -X POST http://localhost:8080/api/v1/poll
+pact -a examples/accounts/scripts/01-system.yaml | curl -H "$JSON" -d @- http://localhost:8080/api/v1/send
+sleep 1
+curl -H "$JSON" -d '{"requestKeys":["020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce"]}' -X POST http://localhost:8080/api/v1/poll
```

Comment:
```suggestion
curl -H "$JSON" -d '{"requestKeys":["db7b34c889d6733ae4201bb7a81c6f3a4ccc7b4289aa4fde4a4eed14126edb7172935a35361ae0a131c3907397a0d0b071d80398e11570c02659510349d9c805"]}' -X POST http://localhost:8080/api/v1/poll
```

Reply:
Hmm we shouldn't be getting different hashes. I will run again but please ensure your command line `pact` is up-to-date.

Reply:
On the branch for me, getting the following hash for 01-system.yaml:
Stuarts-MacBook-Pro-3:pact stuart$ .stack-work/dist/x86_64-osx/Cabal-2.2.0.1/build/Pact/pact -a examples/accounts/scripts/01-system.yaml 
{"cmds":[{"hash":"020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce","sigs":[{"addr":"ba54b224d1924dd98403f5c751abdd10de6cd81b0121800bf7bdbdcfaec7388d","sig":"dbfd6707da51ad2bb8e08840cfc06bbe93cf911c9bb3d7dda11cda140b338871f654520d41ff13aa7a73f4d01c843908c759cddaf9b160a0485b846627bd0b0d","scheme":"ED25519","pubKey":"ba54b224d1924dd98403f5c751abdd10de6cd81b0121800bf7bdbdcfaec7388d"}],"cmd":"{\"payload\":{\"exec\":{\"data\":{\"accounts-admin-keyset\":[\"ba54b224d1924dd98403f5c751abdd10de6cd81b0121800bf7bdbdcfaec7388d\"]},\"code\":\"(define-keyset 'k (read-keyset \\\"accounts-admin-keyset\\\"))\\n(module system 'k\\n  (defun get-system-time ()\\n    (time \\\"2017-10-31T12:00:00Z\\\")))\\n(get-system-time)\"}},\"meta\":{\"gasLimit\":0,\"chainId\":\"\",\"gasPrice\":0,\"sender\":\"\"},\"nonce\":\"\\\"step01\\\"\"}"}]}

Reply:
Yep just rebuilt with your branch and I'm also getting `020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce` for 01-system.yaml.
I'm getting the following hashes:
```Step 1

{"requestKeys":["020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce"]}
Failed reading: satisfy
Step 2

{"requestKeys":["fa6491227ea73ac3cbe00e4b95b9628217ace7a8911e1165caf37440936869121678841eb3d2c2741ba71e4222357b7a853944245a59dcd3bdd4a52a49757c53"]}
{}
Step 3

{"requestKeys":["b31faa1e98ec1b9a2c489f0c5ea019fcf985aab27c46bb778c3bff5fb0e92da3cb67f54004bcd7cb95831c89d4487c9fd84f7f6fa84c64d2e3916afae7dafa3e"]}
{}
Step 4

{"requestKeys":["ec36c9d3dc51fe138b29d9a420b60b82a09bfbb129dd4d8fe2d154799b3a3694fd120aa02bed068f460e61954e8ba1dff7640d439b615c492bf312e2ab8ad5ec"]}
{}```